packages/app/obojobo-repository/shared/components/repository-nav.scss

```diff
@@ -119,6 +121,19 @@
 		width: 2.9em;
 		margin: 0;
 	}
+
+	.notification-indicator {
```

Comment:
Wherever this indicator ends up on the page, make sure there's also a `cursor: pointer;` in there so it's obvious that it can be clicked.